src/content/peerconnection/multiple/js/main.js

```diff
@@ -80,67 +68,33 @@ function call() {
     console.log(`Using video device: ${videoTracks[0].label}`);
   }
   // Create an RTCPeerConnection via the polyfill.
-  const servers = null;
-  pc1Local = new RTCPeerConnection(servers);
-  pc1Remote = new RTCPeerConnection(servers);
-  pc1Remote.ontrack = gotRemoteStream1;
-  pc1Local.onicecandidate = iceCallback1Local;
-  pc1Remote.onicecandidate = iceCallback1Remote;
+  pc1Local = new RTCPeerConnection();
+  pc1Remote = new RTCPeerConnection();
+  pc1Remote.ontrack = e => gotRemoteStream(e, video2);
   console.log('pc1: created local and remote peer connection objects');
 
-  pc2Local = new RTCPeerConnection(servers);
-  pc2Remote = new RTCPeerConnection(servers);
-  pc2Remote.ontrack = gotRemoteStream2;
-  pc2Local.onicecandidate = iceCallback2Local;
-  pc2Remote.onicecandidate = iceCallback2Remote;
+  pc2Local = new RTCPeerConnection();
+  pc2Remote = new RTCPeerConnection();
+  pc2Remote.ontrack = e => gotRemoteStream(e, video3);
   console.log('pc2: created local and remote peer connection objects');
-
-  localStream.getTracks().forEach(track => pc1Local.addTrack(track, localStream));
-  console.log('Adding local stream to pc1Local');
-  pc1Local
-      .createOffer(offerOptions)
-      .then(gotDescription1Local, onCreateSessionDescriptionError);
-
-  localStream.getTracks().forEach(track => pc2Local.addTrack(track, localStream));
-  console.log('Adding local stream to pc2Local');
-  pc2Local.createOffer(offerOptions)
-      .then(gotDescription2Local, onCreateSessionDescriptionError);
-}
-
-function onCreateSessionDescriptionError(error) {
-  console.log(`Failed to create session description: ${error.toString()}`);
+  localStream.getTracks().forEach(track => {
```

Comment:
looks fine to me, but there is no changes in tests, do the tests work the same way after replacing callbacks with async/await?

Reply:
This PR is just refactoring from old style to new style so if the old code was well tested then it will have ensured that this PR did not regress any behavior. Not sure how well tested webrtc samples is in general but what this PR is doing is LGTM